lua/tardis/parts/legacy/legacy_keyboard.lua

```diff
@@ -0,0 +1,18 @@
+--Legacy Interior - Keyboard
+
+local PART = {}
+PART.ID = "legacy_keyboard"
+PART.Name = "Legacy Keyboard"
+PART.Model = "models/drmatt/tardis/keyboard.mdl"
+PART.AutoSetup = true
+PART.Collision = true
+PART.Animate = true
+
+if SERVER then
```

Comment:
Remove section

Reply:
Actually looks like this used to be used to activate advanced mode in legacy, could something similar be applied here? https://github.com/MattJeanes/TARDIS/blob/legacy/lua/entities/sent_tardis_keyboard/init.lua